### ISSUE TITLE
Download offline installers and optional content

### ISSUE THREAD

**Issue by Se3ek** [Heroic-Games-Launcher/HeroicGamesLauncher#3780]
### Problem description

Disclosure: I feel like I'm making a duplicate issue of #1279 but that one has two completely different requests in it and this one that I'm interested in got somewhat buried so I'll try to break it up. Feel free to close it if I'm overstepping.

GOG offers offline installers for all their games and the nerd that I am I like to download those and back them up on my NAS in case I need them ever. GOG Galaxy does a really nice job of letting you download them even to a specific folder that is different from the main game download location. In my case I set this folder to the correct folder on my NAS so the files (sometimes huge obviously) never even hit my internal drive.
I'd love this feature in Heroic too or at least make downloads visible that are executed from the gog embedded browser store.

### Feature description

On the game entry in the library have the option to download optional content like the Offline installers and soundtracks etc.
Have an option in the settings to set a different download directory for those from the games folder.

### Alternatives

If you can't or don't want to implement this, at least have a download progress for files being downloaded that are not triggered from the library.
Context: If you go to the GOG store page in Heroic you can simply got to the library there like in any browser and trigger the download of those elements. They do download without problem but you don't get a progress bar or notification when it's finished or anything. You basically have to hang out until its finished which is only visible when the files don't grow in the target folder anymore.

**Comment by s38b35M5** [comment]
In addition, it seems that the offline game installers download is not using the same method (or number) of connection(s) as when the game is being installed from library. Downloading an offline installer comes down at about 30Mbps for me, and the same game installing from library comes down over 100Mbps, as does the installer when downloaded through the browser. 

It seems problematic that  you can't view the progress or even cancel the download gracefully.

**Comment by dariusmiles2123** [comment]
Agreed. It would be a nice feature to have for game preservation purposes since downloading the game through Heroic and moving it to a safe storage location apparently doesn’t mean you’ll still be able to play the game in 20 years if GOG goes down.
